src/main/java/com/mindsnacks/zinc/classes/data/ZincManifest.java

```diff
@@ -165,14 +164,14 @@ public String getFilePath() {
 
         public String getHashWithExtension() {
             if (isGzipped()) {
-                return getHash() + "." + GZIPPED_FORMAT;
+                return getHash() + "." + FileHelper.GZIPPED_FORMAT;
             } else {
                 return getHash();
             }
         }
 
         public boolean isGzipped() {
-            return mFormats.containsKey(GZIPPED_FORMAT);
+            return mFormats.containsKey(FileHelper.GZIPPED_FORMAT);
```

Comment:
mmh this should actually be called something like `isGzipVersionAvailable` instead of `isGzipped`, right? Don't change in this PR though, I just got confused by the `containsKey`